src/geophires_x/Reservoir.py

```diff
@@ -74,10 +74,9 @@ def __init__(self, model: Model):
 
         self.Tmax = self.ParameterDict[self.Tmax.Name] = floatParameter(
             "Maximum Temperature",
-            value=400.0,
             DefaultValue=400.0,
```

Comment:
Note removal of `value` here is enabled by fix of https://github.com/NREL/GEOPHIRES-X/issues/120 also included in this PR - causing `value` to be set to `DefaultValue` if not explicitly provided.